policies/operators.rego

```diff
@@ -1,5 +1,9 @@
 package clouditor
 
+
+operator = data.operator
```

Comment:
Are you sure, that this is necessary? Didn't it work before?

Reply:
I should add a comment there. It is not for the operator functions, actually. They are declared so that `operator` and `target_vlaue` are added to the output of each policy (I did not want to add these two lines for each rego policy). Otherweise, the assessment can not get these two properties from the evaluation since they are simply not there.